### PR TITLE
HHH-12799 Enforce version alignment of Mockito and ByteBuddy dependen…

### PR DESCRIPTION
**Pull request by Sanne**
…cies

- https://hibernate.atlassian.net/browse/HHH-12799

### CODE REVIEW

File: gradle/libraries.gradle [hunk 1]
```diff
@@ -59,6 +59,7 @@ ext {
 
             // Byte Buddy
             byteBuddy:      "net.bytebuddy:byte-buddy:${byteBuddyVersion}",
+            byteBuddy_agent: "net.bytebuddy:byte-buddy-agent:${byteBuddyVersion}",
```
Comment (gsmet) [inline gradle/libraries.gradle]: AFAICS, you don't use that one.

Comment (Sanne) [reply gradle/libraries.gradle]: I don't but it's good to have it defined as a memo - it's pulled in transitively from Mockito.

Comment (gsmet) [reply gradle/libraries.gradle]: Well, my issue with it is that we will suppose it's what sets the version but it's not at all.

Feel free to push if you think I worry for nothing but I don't like having things misleading the developer. A comment saying that the version of byte-buddy-agent is enforced below would have my preference.

Again, feel free to ignore me on this one :).

Comment (Sanne) [reply gradle/libraries.gradle]: np, I'll remove it since you think that's clearer.

**Comment by Sanne** [comment]
merged

**Pull request closed without merging**